Makefile

```diff
@@ -19,74 +19,50 @@ docker_release_image_cpp_repository := $(docker_image_repository)-cpp
 docker_release_image_python_repository := $(docker_image_repository)-python
 docker_release_image_jupyter_repository := $(docker_image_repository)-jupyter
 
-jupyter_notebook_image_repository := jupyter/scipy-notebook:latest
+jupyter_notebook_image_repository := jupyter/scipy-notebook:python-3.8.8
 jupyter_notebook_port := 9003
-
-project_name_camel_case := $(shell P=$(project_name); echo $${P^})
+jupyter_python_version := 3.8
+jupyter_project_name_python_shared_object := OpenSpaceToolkitMathematicsPy.cpython-38-x86_64-linux-gnu
```

Comment:
Since our "base" Python version is `3.10`, maybe we could use it here too?

Reply:
Yeah I was thinking about this, we would have to update the jupyter notebook version and confirm that everything works. I wanted to save that for a separate MR as we've expanded scope here quite a bit.